DPGAnalysis/SiStripTools/test/detidselectortest_cfg.py

```diff
@@ -63,9 +63,7 @@
 #process.detidselectortest.selections.extend(OccupancyPlotsStripWantedSubDets)
 #process.detidselectortest.selections.extend(OccupancyPlotsPixelWantedSubDets)
 
-process.DQMStore = cms.Service("DQMStore")
-process.TkDetMap = cms.Service("TkDetMap")
-process.SiStripDetInfoFileReader = cms.Service("SiStripDetInfoFileReader")
```

Comment:
Are you sure this line is not needed anymore? In the code of the plugin which is used in this cfg the Service SiStripDetInfoFileReader is used: https://github.com/cms-sw/cmssw/blob/master/DPGAnalysis/SiStripTools/plugins/DetIdSelectorTest.cc#L128

Reply:
The SiStripDetInfoFileReader service is always needed for getting a `TkDetMap`, so I added the corresponding line to (https://github.com/cms-sw/cmssw/pull/21421/files#diff-ef987639bba69bc62be9edeb8d368878)[TkDetMap_cff.py], to avoid copying the same lines in many places. Similarly, (https://github.com/cms-sw/cmssw/pull/21421/files#diff-a5159553817ab5fcc8ee8dc102b1da5d)[TkHistoMap_cff.py], which is included here, includes `TkDetMap_cff.py` and the `DQMStore` line, because the `TkHistoMap` implementation needs it.
I missed that it's also needed directly in this case, I can add the SiStripDetInfoFileReader line back, as you prefer (`cmsRun DPGAnalysis/SiStripTools/test/detidselectortest_cfg.py globalTag=80X_dataRun2_Prompt_v14` runs without errors for me).

Reply:
OK, thanks! No need to add it explicitly then. (I may try to get rid of it in the code, eventually)